tests/unit/services/phar/UnixoidPharInstallerTest.php

```diff
@@ -24,7 +24,7 @@
 use PHPUnit_Framework_MockObject_MockObject;
 
 /**
- * @requires OSFAMILY Linux|Darwin
```

Comment:
Only `OS` supports a regex as a value: https://phpunit.readthedocs.io/en/latest/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests-requires-tables-api

Reply:
So I guess we have to rework this to use `OS` as now this is run on Linux only. I really need a `!Windows` kind of thing ;)